src/crystal/init_cli.cr

```diff
@@ -0,0 +1,141 @@
+require "ecr/macros"
+require "option_parser"
+
+module Crystal
+  module InitCli
+    def self.run(args)
+      options = {} of Symbol => String
```

Comment:
Instead of using a Hash, maybe you could do `config = Config.new` and let config have properties instead of just getters. Then the option parser would set those properties and pass it directly to `InitProject`. That way we avoid an intermediate Hash.

(I don't think this will affect performance at all, the Hash is small, but using `Config` directly will make things a bit more type safe... and in any case all of its properties are nilable)